webui/react/src/pages/Dashboard.tsx

```diff
@@ -74,7 +88,7 @@ const Dashboard: React.FC = () => {
   /* Recent Tasks */
 
   const showTasksSpinner = (
-    !experiments.hasLoaded ||
+    !experiments ||
```

Comment:
would we need to reset experiments to undefined (if we're relying on that for the spinner) when the filters change?

:thinking:  if we could keep using the useRestApi hook it would cover this for us

Reply:
To use the `useRestApi` would be a bit tricky to figure out, but I've added a line to clear out the experiments when the filters change.